resources/assets/js/app/controllers/IdeaHubController.js

```diff
@@ -61,6 +68,32 @@ module.exports = Marionette.Object.extend({
         app.render(view);
 
     },
+    createVote: function () {
+        var VoteAnswers = new VoteAICollection([{name: ''}], {parentUrl: ''});
+        var UsersCollection = new usersCollection();
+        var accessedUsers = new usersCollection();
+
+        UsersCollection.fetch();
+
+        UsersCollection.opposite = accessedUsers;
+        UsersCollection.glyph = 'plus';
+        accessedUsers.opposite = UsersCollection;
+        accessedUsers.glyph = 'minus';
+
+        var model = new VoteModel({user_id: currentUser.get('id')});
+        var view = new CreateVote({
+            model: model,
+            collection: VoteAnswers,
+            users: UsersCollection,
+            accessedUsers: accessedUsers
+        });
+
+        view.listenTo(Radio.channel('votesChannel'), 'createEmptyVoteItem', function (col) {
+            col.add(new VoteAImodel({name: ''}));
```

Comment:
col.add(new VoteAImodel({name: ''}));

I guess that better to define name: '' in model 'defaults'.